kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/MctsBeneficiaryUpdateServiceImpl.java

```diff
@@ -226,7 +189,7 @@ private MctsBeneficiary beneficiaryFromId(String mctsId, String stateMctsId) {
             beneficiary = mctsChildDataService.findByBeneficiaryId(mctsId);
         }
 
-        // then try the State ID
+        // then try the State ID // What on earth is a stateMctsId??
```

Comment:
Don't you want to look into it & find out?
Where's your sense of adventure, man?